sat/cli/bootsys/mgmt_power.py

```diff
@@ -239,13 +240,20 @@ def finish_shutdown(hosts, username, password, ncn_shutdown_timeout, ipmi_timeou
         LOGGER.warning('Forcibly powering off nodes: %s', ', '.join(pending_hosts))
```

Comment:
This line is printed before the user is prompted, so technically we are not forcing the power off yet. We should change it to just say which nodes did not successfully complete a graceful power off within the given timeout. Something like:

```suggestion
        LOGGER.warning('The following nodes did not complete a graceful '
                       'shutdown within the timeout: %s', ', '.join(pending_hosts))
```